lib/semian/sysv_integer.rb

```diff
@@ -0,0 +1,12 @@
+module Semian
+  module SysV
+    class Integer < Semian::Simple::Integer #:nodoc:
```

Comment:
Why are you inheriting from `Simple::Integer`? Won't the implementation be completely different?